src/styles/blocks/catalog.scss

```diff
@@ -11,6 +11,8 @@
     justify-content: center;
     align-items: center;
     box-sizing: border-box;
+    transition-duration: 0.3s;
```

Comment:
would be good to move duration to variable as well

Reply:
the prev comment
<img width="748" alt="image" src="https://user-images.githubusercontent.com/102411130/230297555-5138a133-1e32-4c11-862d-8af19f25a1f4.png">